ludwig/modules/fully_connected_modules.py

```diff
@@ -48,9 +55,12 @@ def __init__(
     ):
         super().__init__()
 
-        #self.layers = []
+        # self.layers = []
```

Comment:
nit: Could you remove the commented out code from this file? Also in lines 106, 121, etc.

Reply:
Done.